addon/services/drag-coordinator.js

```diff
@@ -5,9 +5,13 @@ import { isEqual } from '@ember/utils';
 
 
 function indexOf(items, a) {
-  return items.findIndex(function (element) {
```

Comment:
@ianpetzer Interesting that the findIndex didn't work in your case. I guess the hasMany proxy array doesn't implement this function.